WeatherToday/WeatherToday/SecondViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  SecondViewController.swift
+//  WeatherToday
+//
+//  Created by 오동규 on 2022/08/28.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    
+    @IBOutlet weak var secondTableView: UITableView!
+        
+    let secondCellIdentifier: String = "secondCell"
+
+    var cities: [City] = []
+    
+    var selectedCountry: String?
+    
+    
+
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        self.title = self.selectedCountry
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.secondTableView.delegate = self
+        self.secondTableView.dataSource = self
+        
+        switch selectedCountry
+        {
+        case "독일":
+            decodingJsonDate(country: "de")
+        case "프랑스":
+            decodingJsonDate(country: "fr")
+        case "이탈리아":
+            decodingJsonDate(country: "it")
+        case "일본":
+            decodingJsonDate(country: "jp")
+        case "한국":
+            decodingJsonDate(country: "kr")
+        default:
+            decodingJsonDate(country: "us")
+        }
+        
+        // Do any additional setup after loading the view.
+    }
+    
+    func decodingJsonDate(country: String) {
+        let jsonDecoder: JSONDecoder = JSONDecoder()
+        guard let dataAsset: NSDataAsset = NSDataAsset(name: country) else {
+            return
+        }
+        
+        do {
+            self.cities = try jsonDecoder.decode([City].self, from: dataAsset.data)
+        } catch {
+            print(error.localizedDescription)
+        }
+        
+        self.secondTableView.reloadData()
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+   
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.cities.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "secondCell", for: indexPath) as! secondTableViewCell
```

Comment:
조금더 안전하게 가면 좋을것 같아요~
```suggestion
       guard let cell = tableView.dequeueReusableCell(withIdentifier: "secondCell", for: indexPath) as? secondTableViewCell  else { return UITableViewCell() }
```